pyccel/ast/utilities.py

```diff
@@ -494,14 +494,14 @@ def collect_loops(block, indices, new_index, language_has_vectors = False, resul
             # Loop over indexes, inserting until the expression can be evaluated
             # in the desired language
             new_level = 0
-            for index in range(-rank,0):
+            for index_depth in range(-rank,0):
```

Comment:
`index_depth` is a great name!

By the way:
```suggestion
            for index_depth in range(-rank, 0):
```